app/services/copy_profiles_to_policies.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# A service class to copy Profile data to new Policy objects
+class CopyProfilesToPolicies
+  class << self
+    def run!
+      create_policies
+    end
+
+    private
+
+    def profiles
+      Profile.canonical(false)
+             .where(external: false)
+             .where.not(account_id: nil)
+             .where(policy_id: nil)
+    end
+
+    def create_policies
+      profiles.find_each do |profile|
+        policy = Policy.create!(Policy.attrs_from(profile: profile))
+        policy.hosts << profile.hosts
+        profile.update!(
+          policy: policy
+        )
+      end
+    end
+  end
```

Comment:
This is a fairly naive implementation, but we have only about 1000 policies in production today.